src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -87,9 +97,10 @@ export class AddressForm extends React.Component<
 
           <Flex flexDirection="column">
             <Input
-              id="AddressForm_addressLine2"
+              id="AddressForm_postalCode"
               placeholder="Add postal code"
               title="Postal code"
+              defaultValue={this.props.defaultValue.postalCode}
```

Comment:
This was included already for name, but none of the others. Seems fine to add them, but let me know.